voidcrew/_DEFINES/overmap.dm

```diff
@@ -6,14 +6,18 @@
 
 // These overmap coords are configured to place it in the top left of the z level
 #define OVERMAP_LEFT_SIDE_COORD 1
-#define OVERMAP_RIGHT_SIDE_COORD OVERMAP_SIZE
+#define OVERMAP_RIGHT_SIDE_COORD (OVERMAP_LEFT_SIDE_COORD + (OVERMAP_SIZE - 1))
 
 #define OVERMAP_NORTH_SIDE_COORD (world.maxy)
-#define OVERMAP_SOUTH_SIDE_COORD OVERMAP_NORTH_SIDE_COORD - (OVERMAP_SIZE - 1)
+#define OVERMAP_SOUTH_SIDE_COORD (OVERMAP_NORTH_SIDE_COORD - (OVERMAP_SIZE - 1))
 
 //Possible ship states
 #define OVERMAP_SHIP_IDLE "idle"
 #define OVERMAP_SHIP_FLYING "flying"
 #define OVERMAP_SHIP_ACTING "acting"
 #define OVERMAP_SHIP_DOCKING "docking"
 #define OVERMAP_SHIP_UNDOCKING "undocking"
+
+// minimum job slots for an initial ship
+#define OVERMAP_INITIAL_SHIP_JOB_SLOT_MINIMUM 5
+#define OVERMAP_INITIAL_SHIP_COST_MAXIMUM 200
```

Comment:
We don't plan on adding coins, do we need a ship cost at all for the starter ship?

Reply:
Ideally there would be a whitelist of ships that are allowed to be the starting ship

Reply:
As long as they have 5 slots, is there really a problem with it selecting any?